### ISSUE TITLE
Docs issue

### ISSUE THREAD

**Issue by vitoordaz** [centrifugal/centrifugo#245]
I think that there is a discrepancy in documentation in this section https://centrifugal.github.io/centrifugo/server/configuration/#custom-admin-and-api-ports

It says that api_port and admin_port are supported, but it looks like those parameters were removed in the new version. Am I right? Is this intended?

PS: I can make a PR for documentation changes.

**Comment by FZambia** [comment]
Yes, thanks! Now there is only `internal_port` option. Really appreciate pr.

**Comment by FZambia** [comment]
```
fz@centrifugo: go run main.go --internal_port=9000 --admin --prometheus --debug
2018-10-29T21:15:32+03:00 |INFO| starting Centrifugo dev (go1.11.1)
2018-10-29T21:15:32+03:00 |INFO| config path: /Users/fz/go/src/github.com/centrifugal/centrifugo/config.json
2018-10-29T21:15:32+03:00 |INFO| pid: 5034
2018-10-29T21:15:32+03:00 |INFO| engine: Memory
2018-10-29T21:15:32+03:00 |INFO| gomaxprocs: 4
2018-10-29T21:15:32+03:00 |WARN| DEBUG mode enabled, see /debug/pprof
2018-10-29T21:15:32+03:00 |INFO| serving websocket, SockJS endpoints on :8000
2018-10-29T21:15:32+03:00 |INFO| serving API, admin, prometheus, debug endpoints on :9000
```

When `internal_port` set  the following endpoints will be served using that port: 

* API
* admin
* prometheus
* debug endpoints

Maybe in future there will be a need to set custom ports for specific endpoints, we can always add custom ports later. But now all internal combined to use this internal port option.

**Comment by FZambia** [comment]
@vitoordaz fixed by 13aa173042663e37985346fab831aa920a515bf3, if you have sth to add to improve docs please feel free to do it. Thanks for pointing on this.
